src/Plot.js

```diff
@@ -681,6 +703,25 @@ export default class Plot extends Viz {
     return arguments.length ? (this._baseline = _, this) : this._baseline;
   }
 
+  /**
+       @memberof Plot
+       @desc Sets the confidence to the specified array of lower and upper bounds.
+       @param {Array} *value*
+       @chainable
+   */
```

Comment:
would you mind putting an example here showing passing both accessors and a static key?